test/index.js

```diff
@@ -141,5 +157,19 @@ describe('index', function() {
       watchHandler();
       assert(fs.readFileSync.calledTwice);
     });
+
+    it('should successfully handle database updates and call specified hook', function() {
+      var hook = sinon.spy(function() {});
+      var opts = {
+        watchForUpdates: true,
+        watchForUpdatesHook: hook,
+      };
+      var lookup = maxmind.openSync(dbPath, opts);
+      assert(lookup.get('2001:230::'));
+      console.log(hook.notCalled);
```

Comment:
Replace `console.log` with `assert` please

Reply:
Oops, artifact of debugging tests :)